packages/hacl-star/hacl-star.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "OCaml API for EverCrypt/HACL*"
+maintainer: "Victor Dumitrescu <victor.dumitrescu@nomadic-labs.com>"
+authors: "Victor Dumitrescu <victor.dumitrescu@nomadic-labs.com>"
+license: "Apache-2.0"
+homepage: "https://hacl-star.github.io/"
+bug-reports: "https://github.com/project-everest/hacl-star/issues"
+depends: [
+  "ocaml" {>= "4.05.0"}
+  "hacl-star-raw"
+  "zarith"
+]
+build: ["dune" "build" "-p" name "-j" jobs]
+run-test: ["dune" "runtest"]
+dev-repo: "git+https://github.com/project-everest/hacl-star.git"
+url {
+  src:
+    "https://github.com/project-everest/hacl-star/raw/master/bindings/ocaml/archive/hacl-star.0.1.tar.gz"
```

Comment:
This is a little sketchy. Could you ensure this archive won't ever change? Could it not be a normal git tag / extra download instead?